<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QDialog" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>401</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 145);</string>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string>            Mail            :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reg_mail">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string>        Nombre         :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reg_nombre">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string>     Contrase√±a      :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reg_password">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string> Cancion Favorita :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reg_cancion">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Registro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registrar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(112, 64, 255);
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>REGISTRARSE!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cerrar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Cancelar </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff545f;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ff545f;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string>          Artista          : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reg_artista">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
